lost/src/test/java/com/mapzen/android/lost/internal/LostApiClientImplTest.java

```diff
@@ -67,26 +61,8 @@
     assertThat(LocationServices.SettingsApi).isNull();
   }
 
-  @Test public void disconnect_shouldUnregisterLocationUpdateListeners() throws Exception {
-    client.connect();
-    LocationServices.FusedLocationApi.requestLocationUpdates(LocationRequest.create(),
-        new LocationListener() {
-          @Override public void onLocationChanged(Location location) {
-          }
-
-          @Override public void onProviderDisabled(String provider) {
-          }
-
-          @Override public void onProviderEnabled(String provider) {
-          }
-        });
-
-    client.disconnect();
-    LocationManager lm = (LocationManager) application.getSystemService(LOCATION_SERVICE);
-    assertThat(shadowOf(lm).getRequestLocationUpdateListeners()).isEmpty();
-  }
-
-  @Test public void isConnected_shouldReturnFalseBeforeConnected() throws Exception {
+  @Test @Ignore("Failing after removal of disconnect_shouldUnregisterLocationUpdateListeners")
```

Comment:
At first I was concerned that calling `client.disconnect()` wasn't working the way it should. But it turns out it was. But because `isConnected_shouldReturnFalseBeforeConnected` was the first test running in the class `LostApiClientImplTest` on Circle CI `tearDown()` was not run yet and the client was still connected from a previous test class.

9ac9e10643718cf3952202352424b05d415baff6 seems to have resolved the issue.